.github/workflows/release.yml

```
@@ -347,117 +344,120 @@
-        working-directory: apps/flipper/android
+        working-directory: apps/${{ matrix.app }}/android
       - run: |
           bundle exec fastlane ${{ github.event.inputs.lane || 'internal' }}
         env:
           PLAY_STORE_CONFIG_JSON: ${{ secrets.PLAYSTORE_ACCOUNT_KEY }}
           DB_URL: ${{ secrets.DB_URL }}
           DB_PASSWORD: ${{ secrets.DB_PASSWORD }}
-        working-directory: apps/flipper/android
+        working-directory: apps/${{ matrix.app }}/android
       - name: Upload Android AAB
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)